### ISSUE TITLE
IA: Current analytics & metrics

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4113]
Based on planning in https://github.com/mozilla/foundation.mozilla.org/issues/4090 and [our working doc](https://docs.google.com/document/d/1Co1uocNdhnxmYNBSR9QAg4BQYckTsFYhr1I6vWSCFhY/edit?usp=sharing).

### Todo (WIP list)
- [x] Summary report of our analytics (Tais has already started this [doc](https://docs.google.com/document/d/1KEv-_YJ8ttsZAq9SCJA9PALUsGsDTxunHw6Onb8NDI8/edit))
- [x] Summary of our page performance
  - example: load times
  - consider different audiences when looking at the performance
- [x] Include recommendations for summaries
- [ ] Talk to Arliss, Steph, and Will for numbers
- [ ] Benchmark of what constitutes a good IA
  - Good resource: https://accessibility.digital.gov/content-design/information-architecture/

cc @taisdesouzalessa

**Comment by beccaklam** [comment]
I'm going to start a doc of questions to guide the analytics report. The goal is to write about 50 questions in a brain dump sort of fashion and then refine for a strategy. Brain dump started here: https://docs.google.com/document/d/1QZXBkmmPjosnb1vutJC3rlkFEhFtZx4v0Qjvx9yGJ0w/edit?usp=sharing

@kristinashu @taisdesouzalessa

**Comment by kristinashu** [comment]
Ok awesome, thank you! 

I'm gonna assume that this ticket will carry over into next sprint, but will be done before All Hands.

**Comment by kristinashu** [comment]
Adding Tais' reference link https://www.nngroup.com/articles/ten-usability-heuristics/. We'll measure this against the other IA work we want to do see if it's doable or not.

**Comment by beccaklam** [comment]
Data studio link here with summary report: https://datastudio.google.com/reporting/00aa37ea-3ffb-473f-b3af-cc381eb31b0e

**Comment by beccaklam** [comment]
Will add load times tomorrow!

**Comment by beccaklam** [comment]
# Recommendations

### Observations:
**Our most popular content is as follows:**
- /en/privacynotincluded/ - 17.37% (of Pageviews)
- /en/ - 11.36%
- /en/artifacts/thimble/ - 3.98%

**After these pages, content consumption distribution falls below 2%:**
- /en/campaigns/thank-you-for-your-response/ - 1.39%
- /en/safety-first/ - 1.34%
- and then a combination of pages from /blog & /campaigns ~each approximately around ~1.22 -0.5% 

**Notable page percentages:**
- /en/advocacy/ - 1.16%
- /en/about/ - 1.12%
- /en/initiatives/ - 1.03%
- /en/fellowships/ - 0.73%
- /en/participate/ - 0.59%
- /en/internet-health/ - 0.55%

Given this data, the Foundation site's homepage (/en/) presents a good opportunity to focus efforts for improvement as it is a high traffic page with a relatively high bounce rate. If we can lower this bounce rate we could capture a significant number of eyeballs (a 10% improvement could mean about 20K more page views).

As well, given the general popularity of /campaign and /blog pages, it may serve us well to surface these page type's hubs so that users can easily access them.

## Clues for improvement
Users who land on our homepage and don't drop off go primarily to the 'About' and 'Initiatives' pages. This may indicate a desire to learn more about what Mozilla does in plain language. Our current homepage strategy relies on users to make their inferences about what we do based on what we show on the page. Hypothesis: If we provide this information (e.g. our mission statement and initiatives) in easy to understand language on the homepage, less people will bounce.

### Navigation matters
Another behavioral observation of users from the homepage is that top level navigation pages are visited more than those not included in navigation. This indicates that the top level navigation is still a primary tool people use to traverse our site. What we put here has a higher chance of getting eye balls. Additionally, the language we use here matters as well. "About" and "Initiatives" may be more familiar terms in a user's mental model of an organization. We should consider the language we use here more carefully in how to guide a user through our site. 

### A template that may already be working
Thirdly, from user testing at Mozfest, we had an hypothesis that the "Initiatives" page may offer a better starting template for our homepage. From looking at the page's analytics, this hunch may be worth acting on as the page has a good bounce rate at 49.64%. When users are coming from the homepage, the page has an even greater through traffic rate at 70%.

Finally, while our site load time isn't terrible, some pages take much longer to load than our site average. We should take load times into consideration when we build pages and optimize images.

### Conclusion
So to recap, our recommendations are that we:
- Go ahead and focus on improving the homepage
- Tell users in plain language the mission of Mozilla on the homepage
- Consider the Initiatives page template and About page content as starting points for the homepage redesign
- Consider the language we use for our top level navigation and how it guides user experience
- Make it easier to access the hubs for /blog and /campaigns
- Consider surfacing our initiatives even more (can we group them and include them in our top level nav?)
- Consider page load times and monitor this metric more closely before we launch pages

**Comment by kristinashu** [comment]
Awesome summary and recommendations! @taisdesouzalessa can you think of anything else? If not, I think just add this to the deck and then it's good to close?!?

**Comment by beccaklam** [comment]
@kristinashu do you want all the content in the slide or just the bullet point summary at the end?

**Comment by kristinashu** [comment]
I think everything is useful!

**Comment by beccaklam** [comment]
Put the content in the deck, still need to make it more _presentable_, but will close this ticket for now! @kristinashu

**Comment by taisdesouzalessa** [comment]
Hey @beccaklam And @kristinashu I reopened this ticket to add some observations. I'll close it again once I post it :).

I think those recommendations are spot on. I just have a few notes:

1. > Users who land on our homepage and don't drop off go primarily to the 'About' and 'Initiatives' pages. This may indicate a desire to learn more about what Mozilla does in plain language.

We have the answer for that from our [qualitative research at MozFest (tasks 6 and 7)](https://docs.google.com/document/d/1-1AOVPNCB6hSLy8vpDNYj-8tBgRrRJrPMylI4RKqMws/edit?usp=sharing) - we learned that users wanted to learn what Mozilla Foundation does and found no answer in our homepage. This insight makes your argument that people have the desire to learn more about what Mozilla does even stronger, I recommend you add the context from that research here.

2. Media coverage and email are big contributors to traffic to `/campaigns` and `/blog` pages, I think it would be useful to add this to the report so the whole system (not only the site, isolated) can be considered when we redo the information architecture. Navigation patterns may be different for a person who enters our site through homepage (direct search) vs a person who lands in a campaign page through a news article.

3. Related to site load time, I would add what a good load time is and how our site or pages compare to that. It makes this information more actionable.

I really like how concise and objective this report is. It makes it easy to focus on the things that really matter. Awesome work, Becca.
